subworkflows/local/binning.nf

```diff
@@ -16,15 +16,15 @@ include { FASTA_BINNING_CONCOCT                 } from '../../subworkflows/nf-co
 workflow BINNING {
     take:
     assemblies           // channel: [ val(meta), path(assembly), path(bams), path(bais) ]
-    reads                // channel: [ val(meta), [ reads ] ]
+    _reads                // channel: [ val(meta), [ reads ] ]
```

Comment:
If this isn't used in the workflow at all, remove from the workflow inputs?

Reply:
You're right, it should not be there.